src/devices/devices-list.ts

```diff
@@ -223,6 +219,18 @@ class ESPHomeDevicesList extends LitElement {
         });
       }
 
+      newList.sort((a, b) => {
+        const a_name = a.friendly_name || a.name;
+        const b_name = b.friendly_name || b.name;
+        return a_name
+          .toLocaleLowerCase()
+          .localeCompare(b_name.toLocaleLowerCase());
+      });
+
+      if (devices.importable) {
+        devices.importable.forEach((d) => newList.unshift(d));
```

Comment:
unshift is very inefficient. Better make a copy instead

newList = [...devices.importable, ...newList]

Reply:
Good to know =)
```suggestion
        newList = [...devices.importable, ...newList];
```